index.bs

```diff
@@ -629,7 +633,7 @@ When an {{XRSession}} is shut down the following steps are run:
 
 The <dfn method for="XRSession">end()</dfn> method provides a way to manually shut down a session. When invoked, it MUST run the following steps:
 
-  1. Let |promise| be [=a new Promise=].
+  1. Let |promise| be [=a new Promise=] in the [=relevant realm=] of this {{XRSystem}}.
```

Comment:
```suggestion
  1. Let |promise| be [=a new Promise=] in the [=relevant realm=] of this {{XRSession}}.
```

Reply:
Oh, good catch! I just merged without noticing this, so I'll go apply the change myself.

Reply:
Fixed in 8408e8824708f76ea9193d34b1791d7748fc7a93